secure-api-gateway-ob-uk-common-obie-datamodel/src/main/java/uk/org/openbanking/datamodel/v4/payment/OBWriteDomesticConsentResponse5Data.java

```diff
@@ -15,23 +15,22 @@
  */
 package uk.org.openbanking.datamodel.v4.payment;
 
-import java.util.ArrayList;
-import java.util.List;
-import java.util.Objects;
-
-import org.joda.time.DateTime;
-import org.springframework.format.annotation.DateTimeFormat;
-
 import com.fasterxml.jackson.annotation.JsonProperty;
 import com.fasterxml.jackson.annotation.JsonTypeName;
-
 import io.swagger.v3.oas.annotations.media.Schema;
 import jakarta.annotation.Generated;
 import jakarta.validation.Valid;
 import jakarta.validation.constraints.NotNull;
 import jakarta.validation.constraints.Size;
+import org.joda.time.DateTime;
+import org.springframework.format.annotation.DateTimeFormat;
+import uk.org.openbanking.datamodel.v4.common.OBReadRefundAccount;
 import uk.org.openbanking.datamodel.v4.common.OBStatusReason;
 
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
```

Comment:
can you fix the import order please?

When you generate make sure you run the Intellij optimise imports on the code produced.

Reply:
Ran it multiple times to the point where this is not changing anymore. This is the order the IDE keeps it now.

Reply:
have you got the ForgeRock code style configured?

Reply:
[ForgeRock-IG.zip](https://github.com/user-attachments/files/17519457/ForgeRock-IG.zip)

Reply:
That can be imported to intellij's Settings->Editor->Code Style panel

Reply:
Here are the broader ForgeRock settings that work pretty well;
Instructions:
https://www.jetbrains.com/help/idea/configuring-code-style.html
[ForgeRock (1).zip](https://github.com/user-attachments/files/17519552/ForgeRock.1.zip)

Reply:
Thanks guys, applied the codestyles and reran optimize

Reply:
Thanks, PR looks good.

I'll hold off approving so we can keep the PR open to test out the new development workflow that we've been discussing.